### ISSUE TITLE
Sim fails accessibility fuzz test.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/membrane-transport#380]
For code review #87:

> - [ ] Does the sim pass an accessibility fuzz test? (run with query parameters `fuzzBoard&ea`)

Testing unbuilt on macOS 15.6 with Chrome 139.0.7258.66, the sim fails with:

```
Affirmation failed:  This Disposable has already been disposed, and cannot be disposed again
affirm	@	affirm.ts:43
dispose	@	Disposable.ts:78
(anonymous)	@	UtteranceQueue.ts:256
removeUtterance	@	UtteranceQueue.ts:256
cancelUtterance	@	UtteranceQueue.ts:424
playContent	@	VoicingToolbarItem.ts:289
(anonymous)	@	VoicingToolbarItem.ts:132
notifyLoop	@	TinyEmitter.ts:223
emit	@	TinyEmitter.ts:195
_notifyListeners	@	ReadOnlyProperty.ts:353
unguardedSet	@	ReadOnlyProperty.ts:297
set	@	ReadOnlyProperty.ts:277
set	@	Property.ts:66
(anonymous)	@	VoicingToolbarItem.ts:224
notifyLoop	@	TinyEmitter.ts:223
emit	@	TinyEmitter.ts:206
emit	@	Emitter.ts:68
handleSpeechSynthesisEnd	@	SpeechSynthesisAnnouncer.ts:761
endListener	@	SpeechSynthesisAnnouncer.ts:700
requestSpeech	@	SpeechSynthesisAnnouncer.ts:735
announce	@	SpeechSynthesisAnnouncer.ts:625
attemptToAnnounce	@	UtteranceQueue.ts:599
stepQueue	@	UtteranceQueue.ts:505
notifyLoop	@	TinyEmitter.ts:223
emit	@	TinyEmitter.ts:206
(anonymous)	@	Sim.ts:467
execute	@	PhetioAction.ts:162
stepSimulation	@	Sim.ts:1095
stepOneFrame	@	Sim.ts:1085
runAnimationLoop	@	Sim.ts:1060
requestAnimationFrame
...
```

**Comment by samreid** [comment]
Good find, thanks!  Aside from solving this issue, @jessegreenberg and I could investigate why fuzzboard is not running on CT.

**Comment by jessegreenberg** [comment]
I ran the sim with `?fuzzBoard` for 4 minutes, and then with `fuzzBoard&disableModals` for 3 minutes.

The assertion is related to Voicing so I ran `fuzzBoard&disableModals&voicingInitiallyEnabled` for 3 minutes. Then I exercised the Voicing Toolbar for a while but never saw this assertion.

@samreid and I looked into why CT isn't running fuzzBoard tests and remembered https://github.com/phetsims/aqua/issues/134, investigation can continue there. Closing.
